src/Symfony/Component/String/Tests/Slugger/AsciiSluggerTest.php

```diff
@@ -44,4 +44,27 @@ public function testSlug(string $expected, string $string, string $separator = '
 
         $this->assertSame($expected, (string) $slugger->slug($string, $separator, $locale));
     }
+
+    public function provideSlugEmojiTests(): iterable
+    {
+        yield [
+            'un-chat-qui-sourit-chat-noir-et-a-tete-de-lion-vont-au-parc-national',
+            'un 😺, 🐈‍⬛, et a 🦁 vont au 🏞️',
```

Comment:
```suggestion
            'un 😺, 🐈‍⬛, et une 🦁 vont au 🏞️',
```

Reply:
To, me it's a lion, not a "head of lion". that why I choose to use `un` and not `une`.

Thanks for finding the typo anyway :)